spec/features/cases/offender_sar_complaint/case_creating_spec.rb

```diff
@@ -248,7 +248,7 @@ def then_basic_details_of_show_page_are_correct(offender_sar_case: nil)
   def then_expect_open_cases_page_to_be_correct(offender_sar_case: nil)
     click_on "Cases"
     expect(open_cases_page).to be_displayed
-    expect(cases_page).to have_content "Branston Registry"
+    expect(cases_page).to have_content "branston registry responding user"
```

Comment:
Should this be capitalised as well like the the text it replaced?

Reply:
It's a different field - I've just used what the factory provided here 🤔